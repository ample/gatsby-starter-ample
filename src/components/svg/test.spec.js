import React from "react"
import renderer from "react-test-renderer"

import { component as SVG, fixtures } from "."

describe("SVG", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<SVG {...fixtures.angle_down} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
