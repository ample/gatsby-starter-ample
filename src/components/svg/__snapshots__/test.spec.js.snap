// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SVG renders correctly 1`] = `
<svg
  height={19}
  viewBox="0 0 448 512"
  width={24}
>
  <path
    d="M442 114H6a6 6 0 01-6-6V84a6 6 0 016-6h436a6 6 0 016 6v24a6 6 0 01-6 6zm0 160H6a6 6 0 01-6-6v-24a6 6 0 016-6h436a6 6 0 016 6v24a6 6 0 01-6 6zm0 160H6a6 6 0 01-6-6v-24a6 6 0 016-6h436a6 6 0 016 6v24a6 6 0 01-6 6z"
  />
</svg>
`;
